index.html

```diff
@@ -9,45 +9,217 @@
+                        <li><button class="btn1">CSS</button></li>
+                        <li><button class="btn1">Javascript</button></li>
+                    </ul>
+                </div>
+                <button class="resume">See Project</button>
+            </div>
+            
+          </div>
+          <!-- end -->
+          <!-- tonic -->
+          <div class="grid-container">
+            <div class="grid-item"><img src="images/tonic.jpg" alt="works 1"></div>
+            <div class="grid-item">
+                <div class="works1">
+                    <h1 class="salutation">Tonic</h1>
+                    <div class="flexf">
+                        <p class="canopy">canopy</p> 
+                        <img src="images/Counter.jpg" alt="counter">
+                        <p class="back">back end dev</p>
+                        <img src="images/Counter.jpg" alt="counter">
+                        <p class="back">2015</p>
+                    </div>
+                </div>
+                <div>
+                    <p class="developer">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+              
+                </div>
+                <div>
+                    <ul>
+                        <li><button class="btn1">HTML</button></li>
+                        <li><button class="btn1">CSS</button></li>
+                        <li><button class="btn1">Javascript</button></li>
+                    </ul>
+                </div>
+                <button class="resume">See Project</button>
+            </div>
+            
+          </div>
+          <!-- end -->
+          <!-- Multi-Post Stories -->
+        <div class="grid-container">
+            <div class="grid-item"><img src="images/multi-post-stories.jpg" alt="works 1"></div>
+            <div class="grid-item">
+                <div class="works1">
+                    <h1 class="salutation">Multi-Post
+                        Stories</h1>
+                    <div class="flexf">
+                        <p class="canopy">canopy</p> 
+                        <img src="images/Counter.jpg" alt="counter">
+                        <p class="back">back end dev</p>
+                        <img src="images/Counter.jpg" alt="counter">
+                        <p class="back">2015</p>
+                    </div>
+                </div>
+                <div>
+                    <p class="developer">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+              
+                </div>
+                <div>
+                    <ul>
+                        <li><button class="btn1">HTML</button></li>
+                        <li><button class="btn1">CSS</button></li>
+                        <li><button class="btn1">Javascript</button></li>
+                    </ul>
+                </div>
+                <button class="resume">See Project</button>
+            </div>
```

Comment:
- [ ] I will also suggest that you style your cards as seen in the Figma design. Per the project requirements, you must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). :100:
- I have noted the changes that need to be made on the screenshots below :+1: 

| Your Design | The Figma Design |
|--------|--------|
| ![image](https://github.com/ausiemass/portfolio-austin/assets/102699958/7db0cc13-4eec-4060-8c02-921f37823599)| ![image](https://github.com/ausiemass/portfolio-austin/assets/102699958/4864763f-248c-4a3d-a205-a3a59e0a00ac)|